redirects.txt

```diff
@@ -230,3 +230,6 @@ purchase/replenishment/flows/compute_date.rst applications/inventory_and_mrp/inv
 purchase/replenishment/flows/purchase_triggering.rst applications/inventory_and_mrp/purchase/products/reordering.rst				# (#814)
 purchase/replenishment/flows/setup_stock_rule.rst applications/inventory_and_mrp/purchase/products/reordering.rst					# (#814)
 purchase/replenishment/multicompany/setup.rst applications/general/multi_companies/manage_multi_companies.rst	# (#814)
+
+legal/others/cla.rst legal/cla.rst
+legal/licenses/licenses.rst legal/licenses.rst
```

Comment:
I'd put this on top of all other redirects because content writers will simply add new redirect rules related to applications at the end of the file.